Source/Core/FeatureDetection.js

```diff
@@ -29,11 +29,12 @@ define([
     function isChrome() {
         if (!defined(isChromeResult)) {
             isChromeResult = false;
-
-            var fields = (/ Chrome\/([\.0-9]+)/).exec(theNavigator.userAgent);
-            if (fields !== null) {
-                isChromeResult = true;
-                chromeVersionResult = extractVersion(fields[1]);
+            if (/Google Inc/.test(theNavigator.vendor)) {
```

Comment:
I don't believe this check is correct, it will not work for all variants of Chrome (Chromium, Android, etc...).  Probably better to change it back and also just add a check for `Edge` to make sure we aren't on Edge masquerading as Chrome.